searchapp/urls.py

```diff
@@ -5,6 +5,6 @@
 
 urlpatterns = [
 
-    url(r'^(?P<city>[a-z]+)/(?P<category>[a-z]+)$', views.search, name='search'),
+    url(r'^search/', views.search, name='search'),
```

Comment:
There is no need to use regex here. When you are using some tokens which expires after sometimes then use something like this. e.g. Forget password link must be expired after sometimes